### PR TITLE
DevWs for Aeotec Group GmbH containing containing Z-Wave Switch Generic

### PR DESCRIPTION
**Pull request by Aeotec-ccheng**

### CODE REVIEW

File: devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy [hunk 1]
```diff
@@ -59,10 +59,13 @@ metadata {
 		fingerprint mfr: "0312", prod: "FF00", model: "FF01", deviceJoinName: "Minoston Outlet", ocfDeviceType: "oic.d.smartplug" //Minoston on/off Toggle Switch
 		fingerprint mfr: "0312", prod: "C000", model: "C003", deviceJoinName: "Evalogik Outlet", ocfDeviceType: "oic.d.smartplug" //Evalogik Outdoor Smart Plug
 		fingerprint mfr: "0312", prod: "FF00", model: "FF03", deviceJoinName: "Minoston Switch" //Minoston Smart On/Off Switch
-		fingerprint mfr: "0312", prod: "C000", model: "C005", deviceJoinName: "Evalogik Outlet", ocfDeviceType: "oic.d.smartplug" //Evalogik Mini Outdoor Smart Plug
+		fingerprint mfr: "0312", prod: "C000", model: "CO05", deviceJoinName: "Evalogik Outlet", ocfDeviceType: "oic.d.smartplug" //Evalogik Mini Outdoor Smart Plug
```
Comment (dsobierajsk) [inline devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy]: Why did you change it?

Comment (Aeotec-ccheng) [reply devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy]: I don't recall making a change to other fingerprints but may have been an accident somewhere or maybe i had an older version of this. I will fix that. Sorry about that. 

Edit - Changed and committed via SmartThings IDE.

**Review by dsobierajsk — APPROVED** [review]

**Review by KKlimczukS — APPROVED** [review]
